android/build.gradle

```diff
@@ -1,10 +1,63 @@
 import com.android.Version
+import groovy.json.JsonSlurper
 
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 
+def getReactNativeVersion() {
+    def packageJsonFile = file("$rootDir/../package.json")
+    if (packageJsonFile.exists()) {
+        def packageJson = new JsonSlurper().parseText(packageJsonFile.text)
+        return packageJson.dependencies["react-native"] ?: packageJson.devDependencies["react-native"]
+    }
+    return null
+}
+
+def reactNativeVersion = getReactNativeVersion()
+
+def isReactNativeVersionBetween = { version, minVersion, maxVersionExclusive ->
+    def versionParts = { v -> v.replaceAll(/[^0-9.]/, '').tokenize('.')*.toInteger() }
```

Comment:
Will it work if user choose a RC version ? In a more general way, is there any gradle package to handle versioning check ?